worlds/musedash/docs/setup_en.md

```diff
@@ -0,0 +1,46 @@
+# Muse Dash Randomizer Setup Guide
+
+## Quick Links
+- [Main Page](../../../../games/Muse%20Dash/info/en)
+- [Settings Page](../../../../games/Muse%20Dash/player-settings)
+
+## Required Software
+
+- Windows 8 or Newer.
+- Muse Dash: [Available on Steam](https://store.steampowered.com/app/774171/Muse_Dash/)
+  - \[Optional\] Just As Planned DLC: [Also Available on Steam](https://store.steampowered.com/app/1055810/Muse_Dash__Just_as_planned/)
+- Melon Loader: [GitHub](https://github.com/LavaGang/MelonLoader/releases/latest)
```

Comment:
I'm not sure if you should add that you need multiple .NET Runtimes for Melon loader or a link to its requirements

Reply:
I believe it should just be .NET 6 now for MelonLoader? Might be useful to include

Reply:
Good point.
- Automated installer requires .net 4.8 installed.
- Melon loader requires .net 6 for this game in particular.
I'll update it so that these are mentioned.

Reply:
I've added the needed .net frameworks. Please check out if it makes sense.